### DISCUSSION TITLE
Remove empty spaces in the player

### DISCUSSION THREAD

**Discussion by stevennt** [videojs/video.js#8040] [Q&A]
Hi all, I was trying to remove the empty spaces in the video player, as in the attached image. Tried some display modes without success. Someone must have done it. How did you solve this issue? 

Thanks for sharing.

![image](https://user-images.githubusercontent.com/32351181/206914815-11196a00-9e7a-49ea-9f7f-b755fbcc7dab.png)

**Comment by mister-ben** [comment]
You're not likely to get this unless you've got explicit dimensions set on the player. See https://videojs.com/guides/layout/ for some helper classes around styling.
